src/VisualStudio/Setup.Dependencies/Roslyn.VisualStudio.Setup.Dependencies.csproj

```diff
@@ -31,6 +31,13 @@
     <NuGetPackageToIncludeInVsix Include="System.Runtime.CompilerServices.Unsafe" />
     <NuGetPackageToIncludeInVsix Include="System.Text.Encoding.CodePages" />
     <NuGetPackageToIncludeInVsix Include="System.Threading.Tasks.Extensions" />
+    <!--
+    Remove these once the integration test machines are updated past 16.6 P1
+    https://github.com/dotnet/roslyn/issues/41888
```

Comment:
Wouldn't we have the same problem next time we need to bump versions of these? It seems that we should keep them.

Reply:
I was doing something silly - we already ship the protocol dlls so those are not required.  Just streamjsonrpc.

But yes any time we update streamjsonrpc, if we're updating past the point where integration tests are running we will need this.  But we don't update streamjsonrpc all that often afaik.

Reply:
We don't update the other packages listed above all that often either ;)

Reply:
Fair.
But this appears to break liveshare in F5 scenarios so I think I can't do this anyway :(